src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  margin-top: 0;
```

Comment:
Task description says:
`Remove all the outer spacing for all the heading and paragraphs.`
Not only margin-top